crates/re_viewer/src/app.rs

```diff
@@ -701,188 +545,64 @@ impl eframe::App for App {
                         }
                     }
                 } else {
-                    wait_screen_ui(ui, &self.rx);
+                    crate::ui::wait_screen_ui(ui, &self.rx);
                 }
             });
+    }
 
-        if re_ui::CUSTOM_WINDOW_DECORATIONS {
-            // Paint the main window frame on top of everything else
-            paint_native_window_frame(egui_ctx);
-        }
+    /// Show recent text log messages to the user as toast notifications.
+    fn show_text_logs_as_notifications(&mut self) {
```

Comment:
This bit kind of feels like it belongs in `ui` as well.

Reply:
Yeah, there are still a bunch of things left to move out. More to come (I hope)